challenges/desafio3.js

```diff
@@ -0,0 +1,10 @@
+db.produtos.updateMany({}, { $set: { avaliacao: NumberInt(0) } });
+
+db.produtos.updateMany(
+  { tags: { $all: ["bovino"] } },
```

Comment:
Panta, como você quer filtrar apenas uma tag específica, _bovino_, não precisaria colocar o `$all`.

```suggestion
  { tags: "bovino"} },
```

Reply:
O mesmo se aplica também na linha 8 com a tag: _ave_